### PR TITLE
Mobcompass rewrite

### PR DESCRIPTION
**Pull request by trv6**

### CODE REVIEW

**Comment by trv6** [comment]
Goofed around with this for an hour or so this morning. @sebyg666 can pull out the parts that will fix the current bugs if he wants to. Compass ended up being pretty gaudy, but the important parts seem to work.

**Comment by sebyg666** [comment]
I absolutly love the changes you made, that compass looks awsome. Gona have to look into how you did that graphic. Im by no means the best coder, i just had an idea and tried to implement it. Now You've touched it up its exactly how i wanted it to be in the first place. Nice work. love how the labels only show if your on the right job ^^

**Comment by trv6** [comment]
It's just a bunch of Wingdings characters lined up. There may be a better way to do that.

File: addons/MobCompass/MobCompass.lua [hunk 1]
```diff
@@ -26,392 +26,222 @@ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
...
+
+    texts = require('texts')
+    config = require('config')
+
+    circle = texts.new('l',s_arrows)
+    circle2 = texts.new('l',s_arrows)
+
+    n = texts.new('Ù',s_arrows)
+    s = texts.new('Ú',s_arrows)
+    w= texts.new('×',s_arrows)
```
Comment (z16) [inline addons/MobCompass/MobCompass.lua:49]: This is bugging me more than it should :\

**Pull request merged**